### PR TITLE
feat: implement save game

### PR DESCRIPTION
**Pull request by DenisOleksiuk**

### CODE REVIEW

**Review by Ferri0 — APPROVED** [review]

File: src/js/components/mainMenu.js [hunk 1]
```diff
@@ -29,7 +31,8 @@ class Menu {
     this.menu.classList.add('menu');
     this.menu.innerHTML = `
     ${this.createMenuItem('Новая игра', 'start')}
-    ${this.createMenuItem('Продолжить', 'continue disabled')}
```
Comment (kattitova) [inline src/js/components/mainMenu.js:54]: why did you delete class "disabled"? when user load game at first time, Continue menu item is not able because game not started yet

**Review by kattitova — CHANGES_REQUESTED** [review]

File: src/js/components/mainMenu.js (hunk 1 above)
Comment (DenisOleksiuk) [reply src/js/components/mainMenu.js:54]: I did it for testing load game

**Review by kattitova — APPROVED** [review]

**Review by IgorAleks88 — APPROVED** [review]
